README.md

```diff
@@ -444,16 +440,108 @@ file `/etc/pam.d/fscrypt` containing:
 auth        required    pam_unix.so
 ```
 
-## Note about stability
+## Backup, restore, and recovery
+
+Encrypted files and directories can't be backed up while they are "locked", i.e.
```

Comment:
Should we give the error message that users would expect to see in this case? That way if a user gets a "Required key not available" error during backup, searching this documentation would show them the section that explains the problem?

Reply:
I've added this as a troubleshooting section which links to this section.